birdhouse/optional-components/test-cowbird-jupyter-access/default.env

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+# All env in this default.env can be overridden by env.local.
+
+# All env in this default.env must NOT depend on any other env.  If they do, they
+# must use single quotes to avoid early expansion before overrides in env.local
+# are applied and must be added to the list of DELAYED_EVAL.
+
+# add any new variables not already in 'VARS' or 'OPTIONAL_VARS' that must be replaced in templates here
+# single quotes are important in below list to keep variable names intact until 'pavics-compose' parses them
+EXTRA_VARS='
+  $COWBIRD_JUPYTER_ACCESS_DIR
+'
+# extend the original 'VARS' from 'birdhouse/pavics-compose.sh' to employ them for template substitution
+# adding them to 'VARS', they will also be validated in case of override of 'default.env' using 'env.local'
+VARS="$VARS $EXTRA_VARS"
+
+THIS_FILE="`realpath "$0"`"
+export COWBIRD_JUPYTER_ACCESS_DIR="`dirname "$THIS_FILE"`/optional-components/test-cowbird-jupyter-access"
+
+export JUPYTERHUB_CONFIG_OVERRIDE="
+c.JupyterHub.services = [
+    {
+        'name': 'service-admin',
+        'api_token': 'admin-token',
+    },
+]
+c.JupyterHub.load_roles = [
+    {
+        'name': 'service-role',
+        'scopes': [
+            'admin:users',
+            'admin:servers',
+            'tokens'
+        ],
+        'services': [
+            # assign the service the above permissions
+            'service-admin',
+        ],
+    }
+]
+
+# Specify a fixed image name/version for the test
+c.DockerSpawner.image_whitelist = {'pavics-workflow-test:210216': 'pavics/workflow-tests:210216'}
+
+c.DockerSpawner.environment['TEST_USER'] = 'testcowbirdjupyter'
+c.DockerSpawner.environment['JUPYTERHUB_USER_DATA_DIR'] = os.environ['JUPYTERHUB_USER_DATA_DIR']
+
+notebook_dir = '/notebook_dir'
+c.DockerSpawner.environment['NOTEBOOK_DIR'] = notebook_dir
+c.DockerSpawner.environment['WORKSPACE_DIR'] = notebook_dir + '/writable-workspace'
```

Comment:
Same for this `/writable-workspace` path.  It is probably already defined in some existing variable.  If not, I'd rather you define a new variable in the `jupyterhub_config.py.template` file, then use it here.

We should not redefine the path here because if a change is made in the  `jupyterhub_config.py.template` file, you will not know about it here.